relay-dynamic-config/src/defaults.rs

```diff
@@ -152,6 +152,7 @@ pub fn add_span_metrics(project_config: &mut ProjectConfig) {
                 ("", "transaction.method"),
                 ("", "transaction.op"),
                 ("", "resource.render_blocking_status"), // only set for resource spans
+                ("", "file_extension"),                  // only set for resource spans
```

Comment:
Had to use `file_extension` instead of `extension` because `extension` has already been used in the indexer db.